test/ci/kokoro/config_generator.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Create a config file for gsutil to use in Kokoro tests.
+# https://cloud.google.com/storage/docs/gsutil/commands/config
+
+GSUTIL_KEY=$1
+API=$2
+
+cat  << EOF
+[Credentials]
+gs_service_key_file = "$GSUTIL_KEY"
+
+[GSUtil]
+test_notification_url = https://bigstore-test-notify.appspot.com/notify
+default_project_id = bigstore-gsutil-testing
+prefer_api = "$2"
```

Comment:
Question: You define API above but use $2, and similarly, you define GSUTIL_KEY and use it in the config, is this occurrence supposed to be $API? or should $GSUTIL_KEY above be $1?

I'm no shell expert, so I could be completely wrong...

Reply:
Good catch, updated line 30 to be `prefer_api = "$API"`